consensus/polybft/stake_manager.go

```diff
@@ -256,6 +242,8 @@ func (s *stakeManager) getTransferEventsFromReceipts(receipts []*types.Receipt)
 
 			var transferEvent contractsapi.TransferEvent
 
+			s.logger.Debug("Transfer event log before parse", "Log", log.Data)
```

Comment:
Remove this, since we solved the bug.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1461/commits/b5951c8cbcae5029aeebfae91f42a8d96762bb4d